lib/config/config.js

```diff
@@ -12,10 +13,21 @@ var logger;
 
 var configList = [];
 
+var runtimeConfig = new Matryoshka({}, 'runtime');
+
 var aggregate;
 
 var isInitialized = false;
 
+// Set up the logger with a nice default before we have access to config;
+// This will be overriden by MAGE once during its instanciation, but
```

Comment:
I see a lot of people do this, but ..
https://en.wiktionary.org/wiki/instanciation